selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/CheSeleniumTestHandler.java

```diff
@@ -10,13 +10,23 @@
  */
 package org.eclipse.che.selenium.core;
 
+import static java.lang.Boolean.parseBoolean;
+import static java.lang.System.getenv;
+import static org.eclipse.che.selenium.core.CheSeleniumSuiteModule.CHE_MULTIUSER_VARIABLE;
+
 import com.google.inject.Module;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.List;
+import org.eclipse.che.selenium.core.annotation.Multiuser;
 import org.eclipse.che.selenium.core.inject.SeleniumTestHandler;
+import org.testng.annotations.ITestAnnotation;
 
 /** @author Anatolii Bazko */
 public class CheSeleniumTestHandler extends SeleniumTestHandler {
+  private static final boolean isMultiuser = parseBoolean(getenv().get(CHE_MULTIUSER_VARIABLE));
```

Comment:
Why not to inject?

Reply:
Because of TestNG calls `transform()` method before `onStart(ISuite suite)` which provides injection.

Reply:
I am wondering then why not to move 
```
    Injector injector = createInjector(getParentModules());
    injector.injectMembers(this);
```
into the constructor then?

Reply:
Wiil try it.